app/src/main/java/org/fossasia/openevent/general/auth/LoginViewModel.kt

```diff
@@ -88,18 +89,22 @@ class LoginViewModel(
             }.doFinally {
                 mutableProgress.value = false
             }.subscribe({
-                mutableRequestTokenSuccess.value = verifyMessage(it.message)
+                mutableRequestTokenSuccess.value = true
```

Comment:
Message from the server should be shown here

Reply:
No message is displayed in case of success. Only the mail sent layout is set to visible.

Reply:
The mail sent layout should display the message from server

Reply:
What is the use of message from the server in case of success? Is there any scenario other than mail sent during success?

Reply:
So that if the server changes it's message, it is not needed to change the string in the app